.github/workflows/typecheck.yml

```diff
@@ -4,17 +4,43 @@ on: [push]
 
 jobs:
   build:
+    services:
```

Comment:
to generate the typed files, pgtyped needs to access the most recent database schema.
to get the most recent database schema, we have to run all the migrations on it.
some of those migration files also touch rethinkdb, so we have to require rethinkdb, too.